.github/workflows/release.yml

```diff
@@ -1,347 +1,356 @@
+        run: |
+          curl -sSLo solc/soljson.js https://github.com/ethereum/solidity/releases/download/v0.8.28/soljson.js
+          node -e "
+          const soljson = require('solc/soljson');
+          const createRevive = require('./target/wasm32-unknown-emscripten/release/resolc.js');
+
+          const compiler = createRevive();
+          compiler.soljson = soljson;
+
+          const standardJsonInput =
+          {
+            language: 'Solidity',
+            sources: {
+              'MyContract.sol': {
+                content: 'pragma solidity ^0.8.0; contract MyContract { function greet() public pure returns (string memory) { return \'Hello\'; } }',
+              },
+            },
+            settings: { optimizer: { enabled: false } }
+          };
+
+          compiler.writeToStdin(JSON.stringify(standardJsonInput));
+          compiler.callMain(['--standard-json']);
+
+          // Collect output
+          const stdout = compiler.readFromStdout();
+          const stderr = compiler.readFromStderr();
+
+          if (stderr) { console.error(stderr); process.exit(1); }
+
+          let out = JSON.parse(stdout);
+          let bytecode = out.contracts['MyContract.sol']['MyContract'].evm.bytecode.object
+          console.log(bytecode);
+
+          if(!bytecode.startsWith('50564d')) { process.exit(1); }
+          "
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: revive-wasm
+          path: |
+            ./target/wasm32-unknown-emscripten/release/resolc.js
+            ./target/wasm32-unknown-emscripten/release/resolc.wasm
+            ./target/wasm32-unknown-emscripten/release/resolc_web.js
+          retention-days: 1
+
+  create-release:
+    needs: [tag, build-linux-all, macos-universal-binary]
+    runs-on: ubuntu-24.04
+    permissions:
+      contents: write
+    outputs:
+      upload_url: ${{ steps.create_release.outputs.result }}
+    steps:
+      - name: Download revive-wasm
+        uses: actions/download-artifact@v4
+        with:
+          name: revive-wasm
+          path: resolc-wasm/
+
+      - name: Download revive-linux
+        uses: actions/download-artifact@v4
+        with:
+          name: revive-linux
+          path: resolc-linux/
+
+      - name: Download revive-macos
+        uses: actions/download-artifact@v4
+        with:
+          name: revive-macos
+          path: resolc-macos/
+
+      - name: create-release
+        uses: softprops/action-gh-release@v2
+        with:
+          body: ${{ needs.tag.outputs.RELEASE_NOTES }}
+          tag_name: ${{ needs.tag.outputs.PKG_VER }}
+          name: ${{ needs.tag.outputs.PKG_VER }}
+          draft: true
+          prerelease: true # doublecheck if we need prerelease here
+          files: |
+            ./resolc-linux/resolc-static-linux
+            ./resolc-macos/resolc-macos
+            ./resolc-wasm/resolc.js
+            ./resolc-wasm/resolc.wasm
+            ./resolc-wasm/resolc_web.js
```

Comment:
I know it is shaving the yak here but I didnt realize earlier. Lets pack up those artifacts:
- Linux, macOS, Windows resolc should be compressed wit xz
- The three emscripten artifacts should be tar.xz into one archive.

Reply:
> Linux, macOS, Windows resolc should be compressed wit xz

I see how 50+Mb binaries are better to be compressed, but I'd probably go with `gzip` instead of `xz`, as it's not a builtin tool in macos.  

For windows (when we'll do it) I'd even go with standard `.zip` archive.